src/Search/Search.js

```diff
@@ -193,23 +221,26 @@ class Search extends Component {
               </div>
             </div>
             <div className="pure-u-md-1-6">{periodSelect}</div>
-            {this.showMissingPrisonersButton() && (
-              <Flag
-                name={['updateAttendanceEnabled']}
-                render={() => (
-                  <MissingButtonContainer>
-                    <Button
-                      buttonColour={BLUE}
-                      onClick={() => history.push(`missing-prisoners`)}
-                      data-qa="missing-prisoners"
-                    >
-                      View missing prisoners
-                    </Button>
-                  </MissingButtonContainer>
-                )}
-                fallbackRender={() => <></>}
-              />
-            )}
+            <ButtonsContainer>
+              {this.showMissingPrisonersButton() && (
+                <Flag
+                  name={['updateAttendanceEnabled']}
+                  render={() => (
+                    <MissingButtonContainer>
+                      <StatsLink
+                        buttonColour={BLUE}
+                        onClick={() => history.push(`missing-prisoners`)}
+                        data-qa="missing-prisoners"
+                      >
+                        View missing prisoners
+                      </StatsLink>
+                    </MissingButtonContainer>
+                  )}
+                  fallbackRender={() => <></>}
+                />
+              )}
+              <StatsLink href="/attendance-reason-statistics">View attendance reason statistics</StatsLink>
```

Comment:
is this logic correct?  Should we be showing the statistics to **all** prisons - I didn't think pay was enabled for everyone.
Might be better for initial iteration to not show the link at all and make it hidden otherwise your changes will hold up all future releases of prisonstaffhub until stats verified etc...

Reply:
Yeah i see what you mean.  The initial idea was to build this without the link but it's in the ACs.  Will have a word with Dom.

Reply:
We've got a handler for this  `updateAttendanceEnabled` so I'll drop it within that.  Will only show if the prison has 'pay' enabled then.